### PR TITLE
Fix for trade issue

### PR DESCRIPTION
**Pull request by aminroosta**

### CODE REVIEW

File: .tern-project [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
```
Comment (arnabk) [inline .tern-project]: @aminroosta Whats this file for?

**Comment by arnabk** [comment]
Approved

File: .tern-project (hunk 1 above)
Comment (aminroosta) [inline .tern-project]: [ternjs](http://ternjs.net/) enables autocomplete and code analyzes for javascript files.
Many editors have plugins that work with it, this file is the project configuration for ternjs.

Comment (arnabk) [inline .tern-project]: @aminroosta This is bit personalized. I use webstorm and it works perfectly fine. It has superior auto complete feature and unfortunately, it is not free. please remove this. We dont need it. For code analysis, we should use eslint. We are already ES6, we should go the ESLINT rout

Comment (aminroosta) [inline .tern-project]: @arnabk Done :-)

**Pull request merged**